app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ForgotPasswordsController < ApplicationController
+  def create
+    user = User.find_by_email(params[:email])
```

Comment:
I remember reading that these fancy-methods going through the `method_missing` in Eloquent Ruby recently. But didn't know they were less efficient. Anyway, it doesn't even save much typing :smiley:
Changed all occurrences. Thanks.